### PR TITLE
#140 [doc] Consistency of addon documentation sections - Requirements

### LINKED ISSUE #140: Consistency of addon documentation sections
Layout of addon documentation
* Guide

 * Release notes
    What's new with a page containing a link to github releases page
    Known issues with a page containing a link to github issues page
    Limitations if any
 * Installation and Uninstallation 
 * User Manual
 * Developer Manual

### PR DESCRIPTION
**Pull request by claudiasima-ro**
Resolves #140 

Change-Id: Ie53495f57f07cdcc1cd1169f73befe165c059353
Signed-off-by: claudiasima <claudia.sima@thalesgroup.com>

### CODE REVIEW

File: plugins/org.polarsys.capella.vp.requirements.doc/html/3_userguide/35_propertiesview.html [hunk 1]
```diff
@@ -57,7 +57,7 @@ <h2 id="Relations_between_elements_stored_in_different_layers">Relations between
 		</ol>
 		<p>
 			<i>Some limitations should be noted for Libraries (see the suitable part of 
-				<a href="/wiki/../help/topic/org.polarsys.capella.vp.requirements.doc/html/4_knownissues/4.1_limitations.html" title="../help/topic/org.polarsys.capella.vp.requirements.doc/html/4_knownissues/4.1_limitations.html">Limitations</a> section).
```
Comment (georgiana-ecobici) [inline plugins/org.polarsys.capella.vp.requirements.doc/html/3_userguide/35_propertiesview.html:60]: need add in associated 35_properties.mediawiki the online documentation (span with class="github-link")

**Review by georgiana-ecobici — CHANGES_REQUESTED** [review]
- html heade
- online doc for limitations

File: plugins/org.polarsys.capella.vp.requirements.doc/html/3_userguide/35_propertiesview.html (hunk 1 above)
Comment (claudiasima-ro) [reply plugins/org.polarsys.capella.vp.requirements.doc/html/3_userguide/35_propertiesview.html:60]: Done

File: plugins/org.polarsys.capella.vp.requirements.doc/html/2_installation/2_installation.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version='1.0' encoding='utf-8' ?><!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+	<head>
+		<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+	</head>
+	<body>
```
[the remark this answers is not in the document]
Comment (claudiasima-ro) [reply plugins/org.polarsys.capella.vp.requirements.doc/html/2_installation/2_installation.html:8]: Done

**Review by georgiana-ecobici — CHANGES_REQUESTED** [review]
The current getting started page can be used as Introduction (as in the template below):

image


https://github.com/claudiasima-ro/capella-requirements-vp/blob/master_doc/plugins/org.polarsys.capella.vp.requirements.doc/html/3_userguide/1_gettingstarted.mediawiki

image

**Review by georgiana-ecobici — CHANGES_REQUESTED** [review]
Need also to change the title to Introduction in html:
https://github.com/claudiasima-ro/capella-requirements-vp/blob/master_doc/plugins/org.polarsys.capella.vp.requirements.doc/html/3_userguide/1_gettingstarted.html

image

Change in MediaWiki also the title to Introduction:
https://github.com/claudiasima-ro/capella-requirements-vp/blob/master_doc/plugins/org.polarsys.capella.vp.requirements.doc/html/3_userguide/1_gettingstarted.mediawiki

**Review by georgiana-ecobici — APPROVED** [review]

**Pull request merged**